stustapay/core/config.py

```diff
@@ -35,12 +35,20 @@ class CoreConfig(BaseModel):
     sumup_affiliate_key: str = "unset"
 
 
+class SEPAConfig(BaseModel):
+    sender_name: str
+    sender_iban: str
+    currency: str = "EUR"  # ISO 4217
+    description: str
+
+
 class CustomerPortalApiConfig(HTTPServerConfig):
     base_url: str
     host: str = "localhost"
     port: int = 8082
     base_bon_url: str
     data_privacy_url: str
+    sepa_config: SEPAConfig
```

Comment:
can we have a default here? so one doesn't need this when just running the core.
and then the sample config would need it.